compiler/luci-micro/.gitignore

```
@@ -0,0 +1,2 @@
+.mbedbuild
+cmake_build/
```

Add end of line here:
```suggestion
cmake_build/

```